SKILL.tsv

```diff
@@ -4048,181 +4048,181 @@ SKILL_20150428_004047	{memo X}Summon Demon type boss{nl}Demon power +#{CaptionRa
 SKILL_20150428_004048	{memo X}Effect applies #{CaptionRatio}# times
 SKILL_20150428_004049	{memo X}Back masking range100{nl}Cannot back mask for 10seconds in target area{nl}Use 1 Piece of Dimension
 SKILL_20150428_004050	{#DD5500}{ol}[Physical] - [Pierce]{/}{/}{nl}Arrow breaks down to multiple pieces and bounces nearby.
-SKILL_20150428_004051	Defend range projectile #{CaptionRatio}# times{nl}Parvis Duration #{CaptionTime}#seconds
+SKILL_20150428_004051	$Blocks ranged projectiles: #{CaptionRatio}#{nl}Pavise Duration: #{CaptionTime}# seconds
 SKILL_20150428_004052	{memo X}Attack #{SkillAtkAdd}#{nl}Duration 15 seconds{nl}Use 3 Ash tree
 SKILL_20150428_004053	{memo X}Attack #{SkillAtkAdd}#{nl}Splash #{CaptionRatio}#{nl}Use 1 Claymore
 SKILL_20150428_004054	{memo X}Attack #{SkillAtkAdd}#{nl}Casting 3 seconds{nl}Use 3 Pine Tree
 SKILL_20150428_004055	{#DD5500}{ol}[Physical] - [Strike]{/}{/}{nl}Explode trap and magic circle to attack enemy.
 SKILL_20150428_004056	{memo X}Attack #{SkillAtkAdd}#{nl}#{CaptionRatio}# arrows{nl}Install range #{CaptionRatio2}#{nl}Use 2 wires
-SKILL_20150428_004057	Evasion -#{CaptionRatio}#{nl}Pointing AI duration 90 seconds
-SKILL_20150428_004058	Detect hiding monsters {nl}Hounding AI duration 90 seconds
-SKILL_20150428_004059	Use #{SpendPoison}# Poison Pot Poison
+SKILL_20150428_004057	$Evasion -#{CaptionRatio}#{nl}Pointing AI Duration: 90 seconds
+SKILL_20150428_004058	$Detects hidden monsters {nl}Hounding AI Duration: 90 seconds
+SKILL_20150428_004059	$Consumes #{SpendPoison}# Poison Pot Poison
 SKILL_20150428_004060	{memo X}Attack #{SkillAtkAdd}#{nl}Duration 30 seconds
 SKILL_20150428_004061	Throw tear gas on target area. Target falls under Darkness.
-SKILL_20150428_004062	Tear gas duration #{CaptionTime}# seconds
+SKILL_20150428_004062	$Tear Gas Duration: #{CaptionTime}# seconds
 SKILL_20150428_004063	{memo X}Attack #{SkillAtkAdd}#{nl}Strike 2 times{nl}Critical rate #{CaptionRatio}#%
 SKILL_20150428_004064	{memo X}Attack #{SkillAtkAdd}#{nl}Bleed Duration #{CaptionTime}# seconds{nl}Use 1 Broadhead Arrow
 SKILL_20150428_004065	{memo X}Attack #{SkillAtkAdd}#{nl}Physical Defense -#{CaptionRatio}#% {nl}Use 1 Bodkin Arrow
 SKILL_20150428_004066	{memo X}Attack #{SkillAtkAdd}#{nl}Duration 10 seconds{nl}Use 1 Magic Arrow
 SKILL_20150428_004067	{memo X}Attack #{SkillAtkAdd}#{nl}Duration 10 seconds{nl}Use 1 Gather Corpse
-SKILL_20150428_004068	Hanging Shot
-SKILL_20150428_004069	Patron Saint Duration #{CaptionTime}# seconds{nl}Trigger rate 50%{nl}Trigger #{CaptionRatio}# times
+SKILL_20150428_004068	$Hanging Shot
+SKILL_20150428_004069	$Guardian Saint Duration: #{CaptionTime}# seconds{nl}Chance on hit: 50%{nl}Charges: #{CaptionRatio}#
```

Comment:
Capitalization is inconsistent: "Guardian Saint Duration" vs "Chance on hit". There are more of those, should either capitalize them all (minus the prepositions/particles), or only the first word.